tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -160,6 +162,10 @@ The following lists the CA root paths on common platforms.
 /etc/ssl/ca-bundle.pem
 ```
 
+**Windows**
+
+Windows does not offer a specific path to CA root, instead, it uses [registry](https://learn.microsoft.com/en-us/windows-hardware/drivers/install/local-machine-and-current-user-certificate-stores) to store certificates. For this reason, we recommend downloading [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) as the CA certificates and pointing at this file for the drivers.
+
```

Comment:
Can we provide more detailed steps? For example, how can I point at this file for the drivers?

Reply:
It's the same as other OS. How to point it is drivers' work and we have already described it in previous section.

Reply:
Got it. Can you please link to that section? You know, people don't always read docs linearly and might skip some sections and thus get frustrated.